waspc/src/Wasp/Generator/ServerGenerator.hs

```diff
@@ -170,6 +170,7 @@ genSrcDir wasp =
     [ [C.copySrcTmplAsIs $ C.asTmplSrcFile [relfile|app.js|]],
       [C.copySrcTmplAsIs $ C.asTmplSrcFile [relfile|server.js|]],
       [C.copySrcTmplAsIs $ C.asTmplSrcFile [relfile|utils.js|]],
+      [C.copySrcTmplAsIs $ C.asTmplSrcFile [relfile|core/EntityValidationError.js|]],
```

Comment:
nice, first Haskell code in Wasp! :D 🎆 🍾

Reply:
Yahoo! 🍾 😆

Reply:
Wohoooo :D